Construct multi-qubit Pauli terms and sums from strings (#984)
rigetti/pyquil: CHANGELOG.md

-   PyQuil now sends “modern” ISA payloads to quilc, which must be of version
    \>= `1.10.0`. Check out the details of `get_isa` for information on how to
    specify custom payloads (@ecpeterson, gh-961).

> -   PauliSums can now be constructed from strings and

Reviewer: ```suggestion
-   `PauliSum` objects can now be constructed from strings via `from_compact_str()`
```

>     PauliTerm.from_compact_str() supports multi qubit strings. Added

Reviewer: ```suggestion
    and `PauliTerm.from_compact_str()` supports multi-qubit strings (@jlbosse, gh-984).
```

>     PauliSum.compact_str() in analogy to PauliTerm.compact_str(). (@jlbosse, gh-984)

Reviewer: ```suggestion
```
